src/main/java/org/jfree/chart/entity/StandardEntityCollection.java

```diff
@@ -139,6 +140,28 @@ public ChartEntity getEntity(double x, double y) {
         return null;
     }
 
+    /**
+     * Returns a collection of entities whose area contain the specified point
+     *
+     * @param x the x coordinate.
+     * @param y the y coordinate.
+     *
+     * @return A Vector of ChartEntity objects, possibly empty, never null
+     */
+    @Override
+    public Vector<ChartEntity> getEntities(double x, double y){

```

Comment:
Is there any order requirement for the returned list?  If not, just return a `Collection`.

Reply:
I can't remember if there is or not - for sure we need the synchronization though. 

Aside from that, the modifications are very close to going into production (after having been delayed multiple times).  And if we were to mess with the release schedule again, it would have a ripple effect - above my pay grade lol. (And this is at the heart of what's annoying about this career, lol - all the non-technical stuff that influences when/how we do our job...)